### PR TITLE
Implement action button on table

### PR DESCRIPTION
**Pull request by marc-aurele-besner**
## Implement an action button on table

Show an Actions dropdown button on the row that matches the operator your wallet own

- #19 

TableWithAction
De-register modal
Withdraw modal

### CODE REVIEW

**Review by EmilFattakhov — COMMENTED** [review]
Hm, somehow I don't see any available actions for the existing and running operator, or for the newly registered one 🤔 

Screenshot 2023-11-15 at 9 07 05 PM
Screenshot 2023-11-15 at 9 09 11 PM

File: src/components/operatorsList.tsx [hunk 1]
```diff
@@ -82,6 +91,13 @@ export const OperatorsList: React.FC<OperatorsListProps> = ({ operatorOwner }) =
                   <Td {...textStyles.text} isNumeric>
                     {hexToFormattedNumber(operator.operatorDetail.currentTotalStake)}
                   </Td>
+                  {isOneOfTheOperators &&
```
Comment (EmilFattakhov) [inline src/components/operatorsList.tsx:94]: I guess something here might not be matching in my case, hence I don't see any available actions

Comment (marc-aurele-besner) [reply src/components/operatorsList.tsx:94]: Thanks I believe I fix this in my last push

**Review by EmilFattakhov — APPROVED** [review]
Works well Marc!

**Pull request merged**